### DISCUSSION TITLE
Jetson Nano Settings

### DISCUSSION THREAD

**Discussion by goldiiii** [blakeblackshear/frigate#1085] [Ask A Question]
Hi guys,

I would like to know which settings are best for the Jetson Nano. Does it support hardware acc (ffmpeg settings)?

**Comment by slacker-zen** [comment]
nvidia provides HWACCEL in their ffmpeg package.

https://github.com/uburuntu/jetson-startup

how do you run frigate on the jetson?

greetings

**Reply by LordNex** [reply]
This is what I'm going to try. But I'm going to use a USB coral TPU for object detection. I'm really hoping that the GPU power can be utilized to decode/encode the streams
